test cases/common/276 generator custom_tgt subdir/meson.build

```diff
@@ -0,0 +1,22 @@
+project('276 generator custom_tgt subdir')
+
+touch = find_program('touch', required: false)
+if not touch.found()
+  error('MESON_SKIP_TEST: touch is required to run the test')
+endif
```

Comment:
I could get rid of this if asked.

Reply:
Something along the lines of
```
stamp_cmd = [find_program('python', required: true), '-c', 'import sys; open(sys.argv[1], "w")', '@OUTPUT0@']
```
might work?

Reply:
I think` 'import sys, os; os.utime(sys.argv[1])` is the way to do this.